spec/lib/phone_number_parser_spec.rb

```diff
@@ -1,23 +1,57 @@
-require "rails_helper"
+require 'rails_helper'
```

Comment:
Yeah, I looked it up after you used single quotes in one of your suggested changes. I'm happy to standardize around single quotes, and'll change them as I touch files for other reasons.